changelog/22.0/22.0.0/changelog.md

```diff
@@ -0,0 +1,5 @@
+# Changelog of Vitess v22.0.0
```

Comment:
I believe this file will be generated when we do the first release candidate for v22. No need to create it right now.